utils/crypto/bls/signer/rpcsigner/client.go

```diff
@@ -17,51 +21,69 @@ var _ bls.Signer = (*Client)(nil)
 
 type Client struct {
 	client pb.SignerClient
-	conn   *grpc.ClientConn
 	pk     *bls.PublicKey
 }
 
-func NewClient(ctx context.Context, conn *grpc.ClientConn) (*Client, error) {
+func NewClient(ctx context.Context, url string) (*Client, func() error, error) {
+	// TODO: figure out the best parameters here given the target block-time
+	opts := grpc.WithConnectParams(grpc.ConnectParams{
+		Backoff: backoff.DefaultConfig,
+	})
```

Comment:
Can we better describe what this is doing? On a quick look - it doesn't seem to do anything. But on closer inspection, it seems to be lowering the `MinConnectTimeout` from `20s` to `0`. Can we explicitly set the `MinConnectTimeout: 0` in the `ConnectParams` and comment that this is why we are specifying the params? It is fairly common for unspecified (0) to be overridden to the default value... But that doesn't seem to happen here.

Alternatively, if the goal wasn't to modify the `MinConnectTimeout` - should we delete this option?